server/server.go

```diff
@@ -1046,6 +1046,15 @@ func validateOptions(o *Options) error {
 		return fmt.Errorf("max_payload (%v) cannot be higher than max_pending (%v)",
 			o.MaxPayload, o.MaxPending)
 	}
+	// Check that account's trace_dest is a valid subject.
+	for _, acc := range o.Accounts {
+		if acc.TraceDest == _EMPTY_ {
+			continue
+		}
+		if !IsValidSubject(acc.TraceDest) {
```

Comment:
`IsValidPublishSubject`?

Reply:
That is a good point and will add to the test that I will reject wildcard subjects (which I assume `IsValidPublishSubject` does.